lib/go-tc/cdns.go

```diff
@@ -83,7 +85,7 @@ type CDNNullable struct {
 	// ID of the CDN
 	//
 	// required: true
-	ID *int `json:"id" db:"id"`
+	impl.ID
```

Comment:
`lib/go-tc/cdns.go:88:2: error: reference to undefined identifier 'impl.ID'`
I think either this should be `impl.ImplID` or you need to rename the struct to `ID`. Either will work, but I'm partial to the latter, myself; `impl.ImplID` feels redundant.